Ruby-Project/student_class.rb

```diff
@@ -0,0 +1,11 @@
+require './person_class'
+class Student < Person
+  def initialize
+    super(classroom)
+    @classroom = classroom
+  end
```

Comment:
- [ ]The constructor extends the parent's constructor by adding @classroom and a parameter. Link to the example about constructors in a previous lesson https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars. For it to work however, the constructor should take all the variables as arguments including the classroom. the classroom shouldn't be passed to the super however because this argument was never created in the person class you should however put the age, name and parent_permission as arguments in super.